src/component.js

```diff
@@ -38,7 +38,7 @@ function createSubmitHandler (hostedFieldsInstance, orderIdFunction) : Function
         logger.track({
           comp:                                'hostedpayment',
           // risk_correlation_id: 'TODO',
-          card_brand:                          payload && payload.payment_source.card.card_type,
+          card_brand:                          payload && payload.payment_source && payload.payment_source.card.card_type,
```

Comment:
I still don't think this is the correct thing to do. Is there anyway to get the card details from the validate call after the 3ds flow completes?